src/Examine.Lucene/Suggest/LuceneSuggestionQueryBase.cs

```diff
@@ -0,0 +1,28 @@
+using Examine.Suggest;
+
+namespace Examine.Lucene.Suggest
+{
+    public abstract class BaseLuceneSuggester : BaseSuggesterProvider
```

Comment:
This is public and should therefore have XML documentation - Use `/// <inheritdoc />` where appropriate

Reply:
Done